runtime/envelope/envelope.go

```diff
@@ -164,7 +166,17 @@ func NewEnvelope(ctx context.Context, wlet *protos.EnvelopeInfo, config *protos.
 	// Create the connection, now that the weavelet is running.
 	conn, err := conn.NewEnvelopeConn(e.ctx, toEnvelope, toWeavelet, e.weavelet)
 	if err != nil {
-		return nil, err
+		if errors.Is(err, syscall.EPIPE) || errors.Is(err, io.EOF) {
```

Comment:
Just a clarifying question, what is special about EPIPE and EOF here? Is it possible for us to always read the contents of the stderr pipe and return it?

Reply:
What happens if the error is not EPIPE or EOF and we try to read errpipe? Does that always terminate? I was thinking that it might be useful to always return the stderr?